src/DeviceInterfaceValues.tsx

```diff
@@ -554,6 +555,12 @@ export default (): React.ReactElement => {
       });
   };
 
+  useEffect(() => {
+    if (iface?.type === 'datastream') {
+      navigate(`/devices/${deviceId}/interfaces/${interfaceName}/datastream`);
```

Comment:
Let's not force users out of this page. Let's display a link instead (maybe in the top right?)